learn-cockroachdb-sql.md

```diff
@@ -48,7 +48,7 @@ To see all databases, use the [`SHOW DATABASES`](show-databases.html) statement:
 
 ## Set the Default Database
 
-To set the default database, use the [`SET DATABASE`](set-database.html) statement:
+To set the default database, use the [`SET`](set-vars.html) statement:
```

Comment:
`set-vars.md` should have a header in its Examples section that this can link to.